components/merkledb/src/views/address.rs

```diff
@@ -1,7 +1,24 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 use std::{borrow::Cow, num::NonZeroU64};
 
 use crate::BinaryKey;
 
+/// Separator between the name and the additional bytes in family indexes.
+pub const INDEX_NAME_SEPARATOR: &[u8] = &[SEPARATOR_CHAR];
```

Comment:
I suggest to have it `pub(super)`: despite the module being private, the strict visibility is easier to understand at first glance.